From 3a5edc5cc42ec7fc3b2b71709b63c974df223d22 Mon Sep 17 00:00:00 2001
From: Stefan Wahren <stefan.wahren@in-tech.com>
Date: Thu, 19 Aug 2021 10:32:35 +0200
Subject: [PATCH 547/552] overlays: qca7000: Fix URL & README

I2SE has been acquired by in-tech, so fix the URL & README accordingly.

Signed-off-by: Stefan Wahren <stefan.wahren@in-tech.com>
---
 arch/arm/boot/dts/overlays/README              | 2 +-
 arch/arm/boot/dts/overlays/qca7000-overlay.dts | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/README b/arch/arm/boot/dts/overlays/README
index 708ebaac9c92..d50d694c9e77 100644
--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -2504,7 +2504,7 @@ Params: gpio_pin                Output GPIO (default 18)
 
 
 Name:   qca7000
-Info:   I2SE's Evaluation Board for PLC Stamp micro
+Info:   in-tech's Evaluation Board for PLC Stamp micro
 Load:   dtoverlay=qca7000,<param>=<val>
 Params: int_pin                 GPIO pin for interrupt signal (default 23)
 
diff --git a/arch/arm/boot/dts/overlays/qca7000-overlay.dts b/arch/arm/boot/dts/overlays/qca7000-overlay.dts
index 9a451202a2eb..f695f36024fa 100644
--- a/arch/arm/boot/dts/overlays/qca7000-overlay.dts
+++ b/arch/arm/boot/dts/overlays/qca7000-overlay.dts
@@ -1,5 +1,5 @@
-// Overlay for the Qualcomm Atheros QCA7000 on I2SE's PLC Stamp micro EVK
-// Visit: https://www.i2se.com/product/plc-stamp-micro-evk for details
+// Overlay for the Qualcomm Atheros QCA7000 on PLC Stamp micro EVK
+// Visit: https://in-tech-smartcharging.com/products/evaluation-tools/plc-stamp-micro-2-evaluation-board for details
 
 /dts-v1/;
 /plugin/;
-- 
2.33.0

